tests/NPSP.py

```diff
@@ -339,8 +351,8 @@ def select_relatedlist(self,title):
         locator=npsp_lex_locators['record']['related']['title'].format(title)
         self.selenium.get_webelement(locator).click()  
         
-    def verify_contact_roles(self, **kwargs):
-        """verifies the role of a specific contact on Opportunities page""" 
+    def verify_related_list_field_values(self, **kwargs):
+        """verifies the values in the related list objects page""" 
         for name, value in kwargs.items():
             locator= npsp_lex_locators['object']['contact_role'].format(name,value)
```

Comment:
This selector doesn't look like it is specific to contact roles and should be renamed to something more generic.

Reply:
thanks for catching it, I changed the keyword name and forgot to change the locator name